api/base/urls.py

```diff
@@ -38,7 +38,8 @@
                 url(r'^institutions/', include('api.institutions.urls', namespace='institutions')),
                 url(r'^licenses/', include('api.licenses.urls', namespace='licenses')),
                 url(r'^logs/', include('api.logs.urls', namespace='logs')),
-                url(r'^metaschemas/', include('api.metaschemas.urls', namespace='metaschemas')),
+                url(r'^metaschemas/', include('api.schemas.urls', namespace='metaschemas')),
```

Comment:
I wasn't sure how to effectively deprecate `/metaschemas/` as a whole URL base without having a whole lot of duplicated code, so I simply left the route there and added a duplicate `/schemas/` base url route as well. I am sure I am missing something here as far as proper deprecation!

Reply:
Check out what we did for `/v2/preprint_providers/` [here](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/preprint_providers/urls.py).

Reply:
Ah, I see what you mean now. I think this is consistent as-is -- we still have `/preprint_providers/` in this file even though all of the sub-routes are deprecated (replaced by `/providers/preprints/`).